migration/src/test/java/org/axonframework/migration/AxonJakartaTest.java

```diff
@@ -0,0 +1,79 @@
+package org.axonframework.migration;
+
+
+import org.junit.jupiter.api.Test;
+import org.openrewrite.java.JavaParser;
+import org.openrewrite.test.RecipeSpec;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+import static org.openrewrite.java.Assertions.mavenProject;
+import static org.openrewrite.maven.Assertions.pomXml;
+
+public class AxonJakartaTest implements RewriteTest {
+
+    @Override
+    public void defaults(RecipeSpec spec) {
+        spec
+                .parser(JavaParser.fromJavaVersion()
+                        .logCompilationWarningsAndErrors(true)
+                        .classpath("axon-messaging", "rewrite-spring"))
+                .recipe("/META-INF/rewrite/axon-jakarta.yml", "org.axonframework.axon.UpgradeAxonFramework_4_7_Jakarta");
+    }
+
+    @Test
+    void migrateImports() {
```

Comment:
Although extremely verbose, shouldn't this validate *all* changes in the YAML?

Reply:
You're welcome to add them all. ;)

Typically though at that point you're mostly testing that there's no typo's by adding duplicates here. The migration recipes themselves are tested outside this project already. It's just the parameters that you might want to cover in case there's mismatches, or classes removed later.

Reply:
> You're welcome to add them all. ;)

Assumed as much! ;-)
Just validating my assumption with the request. I don't fancy pushing people to put in tedious work (however much appreciated it might be).

> The migration recipes themselves are tested outside this project already.

With this, do you mean "manually" validation on a sample project?

Reply:
> With this, do you mean "manually" validation on a sample project?

No I meant there comes a point where you might end up testing whether OpenRewrite can replace classes/imports; we have that covered on our end. :)

Reply:
Gotcha, makes sense